### PR TITLE
Aws

### PR DESCRIPTION
**Pull request by trezorg**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+polcadot tests
```
Comment (ArseniiPetrovich) [inline tests/README.md:1]: It's polKadot, not polCadot

File: tests/gcp/bundle_test.go [hunk 2]
```diff
@@ -1,321 +1,145 @@
...
-            "prefix": prefix,
-            "delete_on_termination": "true",
-            "cpu_limit": "1",
-            "ram_limit": "1",
-	    "validator_name": "test",
-	    "expose_ssh": "true",
-	    "node_key": "fc9c7cf9b4523759b0a43b15ff07064e70b9a2d39ef16c8f62391794469a1c5e",
-            "chain": "westend",
-	    "admin_email": "1627_DEV@altoros.com", 
+			"aws_regions":           helpers.BuildRegionsParam(gcpRegion...),
```
Comment (ArseniiPetrovich) [inline tests/gcp/bundle_test.go:46]: aws_regions in GCP? Probably a mistake here.

**Review by ArseniiPetrovich — CHANGES_REQUESTED** [review]
I like the changes this PR introduces, but there are some comments that should be addressed before merge.

File: tests/gcp/bundle_test.go (hunk 2 above)
Comment (trezorg) [reply tests/gcp/bundle_test.go]: Fixed

File: tests/README.md (hunk 1 above)
Comment (trezorg) [reply tests/README.md]: Fixed

**Review by ArseniiPetrovich — APPROVED** [review]

**Pull request merged**